plugins/aws/fix_plugin_aws/collector.py

```diff
@@ -271,7 +262,7 @@ def get_last_run() -> Optional[datetime]:
                     log.warning(f"Unexpected node type {node} in graph")
                     raise Exception("Only AWS resources expected")
 
-            access_edge_collection_enabled = os.environ.get("ACCESS_EDGE_COLLECTION_ENABLED", "false").lower() == "true"
+            access_edge_collection_enabled = False
```

Comment:
This basically disables the functionality always. Let's use it this way: 
- Remove this flag
- config defaults to false
- The config property has a warning as a description.

If someone wants to turn it on, it must be enabled in the configuration with clear documentation.